src/bootstrap/test.rs

```diff
@@ -924,6 +924,7 @@ impl Step for RustdocGUI {
                     .arg(&out_dir)
                     .env("RUSTDOC", builder.rustdoc(self.compiler))
                     .env("RUSTC", builder.rustc(self.compiler))
+                    .env("RUSTDOCFLAGS", "-Zunstable-options --generate-link-to-definition")
```

Comment:
Seems somewhat unfortunate to enable `--generate-link-to-definition` for *all* GUI tests. I guess it could have a `compile-flags` comment like compiletest does, but that would be a larger change.

Reply:
Actually I think I'll enable it only for one test to simplify things.